### PR TITLE
CNV-37398: Wrong Notes about Node Health Check in Cluster high-availability options section

### PR DESCRIPTION
**Pull request by sbeskin-redhat**
Resolves: https://issues.redhat.com/browse/CNV-37398

OCP 4.12+

Preview: https://72295--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/install/preparing-cluster-for-virt#cluster-high-availability-options_preparing-cluster-for-virt

### CODE REVIEW

File: virt/install/preparing-cluster-for-virt.adoc [hunk 1]
```diff
@@ -158,14 +158,6 @@ In {product-title} clusters installed using installer-provisioned infrastructure
 ====
 
 * Automatic high availability for both IPI and non-IPI is available by using the *Node Health Check Operator* on the {product-title} cluster to deploy the `NodeHealthCheck` controller. The controller identifies unhealthy nodes and uses the Self Node Remediation Operator to remediate the unhealthy nodes. For more information on remediation, fencing, and maintaining nodes, see the link:https://access.redhat.com/documentation/en-us/workload_availability_for_red_hat_openshift/23.2/html-single/remediation_fencing_and_maintenance/index#about-remediation-fencing-maintenance[Workload Availability for Red Hat OpenShift] documentation.
```
Comment (anna-savina) [inline virt/install/preparing-cluster-for-virt.adoc:160]:
```suggestion
* Automatic high availability for both IPI and non-IPI is available by using the *Node Health Check Operator* on the {product-title} cluster to deploy the `NodeHealthCheck` controller. The controller identifies unhealthy nodes and uses the remediation provider (for example Self Node Remediation Operator or Fence Agents Remediation  Operator) to remediate the unhealthy nodes. For more information on remediation, fencing, and maintaining nodes, see the link:https://access.redhat.com/documentation/en-us/workload_availability_for_red_hat_openshift[Workload Availability for Red Hat OpenShift] documentation.
```

**Comment by anna-savina** [comment]
Better to use a link to the start page of Workload Availability for Red Hat OpenShift documentation, then you don't need to update for each new version
https://access.redhat.com/documentation/en-us/workload_availability_for_red_hat_openshift

**Comment by sbeskin-redhat** [comment]
> Better to use a link to the start page of Workload Availability for Red Hat OpenShift documentation, then you don't need to update for each new version https://access.redhat.com/documentation/en-us/workload_availability_for_red_hat_openshift

Done

**Review by Srivaralakshmi — COMMENTED** [review]
@sbeskin-redhat Neat PR, Good job! LGTM. Thank you!

**Pull request merged**